File: app/src/main/java/com/github/chagall/notificationlistenerexample/SQLDatabaseHelper.java
package com.github.chagall.notificationlistenerexample;

import android.content.Context;
import android.database.Cursor;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;
import android.database.sqlite.SQLiteQueryBuilder;
import android.util.Log;

import com.readystatesoftware.sqliteasset.SQLiteAssetHelper;

public class SQLDatabaseHelper extends SQLiteAssetHelper {

    private static final String DATABASE_NAME = "disinfo_domains.db";
    private static final int DATABASE_VERSION = 1;

    public SQLDatabaseHelper(Context context) {
        super(context, DATABASE_NAME, null, DATABASE_VERSION);
    }

    public Cursor getDisinfoLinks() {

        SQLiteDatabase db = getReadableDatabase();
        SQLiteQueryBuilder qb = new SQLiteQueryBuilder();

        String [] sqlSelect = {"domain"};
        String sqlTables = "disinfo_links";

        qb.setTables(sqlTables);
        Cursor c = qb.query(db, sqlSelect, null, null,
                null, null, null);

        c.moveToFirst();
        return c;

    }
}

File: app/src/main/java/com/standford/ligthhouse/db/SQLDatabaseHelper.kt
package com.standford.ligthhouse.db

import android.content.Context
import android.database.Cursor
import android.database.sqlite.SQLiteQueryBuilder
import com.readystatesoftware.sqliteasset.SQLiteAssetHelper

class SQLDatabaseHelper(context: Context?) :
    SQLiteAssetHelper(context, DATABASE_NAME, null, DATABASE_VERSION) {
    val disinfoLinks: Cursor
        get() {
            val db = readableDatabase
            val qb = SQLiteQueryBuilder()
            val sqlSelect = arrayOf("domain")
            val sqlTables = "disinfo_links"
            qb.tables = sqlTables
            val c = qb.query(
                db, sqlSelect, null, null,
                null, null, null
            )
            c.moveToFirst()
            return c
        }

    companion object {
        private const val DATABASE_NAME = "disinfo_domains.db"
        private const val DATABASE_VERSION = 1
    }
}